.github/workflows/test_defender_function.yaml

```diff
@@ -0,0 +1,203 @@
+# Purpose: Run the product functional tests every night
+# The testing params are stored as a string in a CSV-like format.
+# Pipes separate products, and commas separate key/value pairs.
+# key1=value1,key2=value2|key3=value3,key4=value4...
+# Test version for Defender only - burn after testing
+
+name: Defender Functional Tests (temp)
```

Comment:
Minor detail:  In order to delete a temp workflow, you have to manually delete each individual run.  Severe GitHub limitation.  Don't keep this around too long.  :-)